src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String [] args) {
+        System.out.println("------------------");
+        Truck truck = new Truck();
+        truck.doWork();
+        Bulldozer bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        Excavator excavator = new Excavator();
+        excavator.doWork();
+        System.out.println("-------------------");
+        excavator.stopWork();
+        truck.stopWork();
+        bulldozer.stopWork();
+        System.out.println("-------------------");
```

Comment:
Remove all `System.out.println("-------------------");`
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop): Use abstract reference instead of specific one where possible
and lets start and spot machines in a loop(we can put them to array for this purpose)